sova/cli/commands/migrate.py

```
@@ -0,0 +1,112 @@
+    import tomllib
+except ImportError:
+    import tomli as tomllib  # type: ignore[no-redef]
+
+
+def migrate_config(
+    project: Annotated[Optional[str], typer.Argument(help="Project directory (defaults to current directory)")] = None,
+    dry_run: Annotated[bool, typer.Option("--dry-run", help="Show what would be migrated without writing")] = False,
+    remove_toml: Annotated[
+        bool, typer.Option("--remove-toml", help="Delete sova.toml after successful migration")
+    ] = False,
+    force: Annotated[
```

Ruff (0.16.2)

[warning] Boolean default positional argument in function definition

(FBT002)